Oneirocom/Magick
# Remove sqlite, postgres only

## What Changed:

This completely removes sqlite, we are only relying on postgres

This also removes a bunch of console.logs because I was getting killed by logspam tracking things down :)

## How to test:

You will need a Postgres client setup. Either run the docker container or paste in a connection string

NOTE: To merge this PR we will need to make sure that there is a default connection string and just-works solution with npm install and npm run dev
